### PR TITLE
Prevent voting in SCHEDULE_PUBLISHED conferences

### PR DESCRIPTION
**Pull request by smtchahal**
Prevent voting only in those proposals whose conferences have the status
SCHEDULE_PUBLISHED.

Relevant: #372

### CODE REVIEW

File: junction/proposals/votes_views.py [hunk 1]
```diff
@@ -113,8 +116,9 @@ def proposal_reviewer_vote(request, conference_slug, proposal_slug):
     proposal = get_object_or_404(Proposal, slug=proposal_slug,
                                  conference=conference)
 
-    if not permissions.is_proposal_section_reviewer(request.user,
-                                                    conference, proposal):
+    if (not permissions.is_proposal_section_reviewer(request.user,
```
Comment (kracekumar) [inline junction/proposals/votes_views.py]: Two `not` can be reduced to one right ?

**Comment by kracekumar** [comment]
Can you pull latest master ?

**Comment by kracekumar** [comment]
Just noticed, can you add test case ?

**Comment by smtchahal** [comment]
Sure. I'm on it.

**Comment by smtchahal** [comment]
Actually, I'm kind of stuck at something and I can't figure it out :disappointed:.

The tests I need to add need access to a proposal that was created in a conference with the status "schedule published" status, but those don't exist in fixtures. So I edited the `create_proposals` function so it returns a dictionary, similar to the `conferences` fixture, returning a `past` and a `future` proposal. Then I wrote the tests and they ran successfully, as expected.

The problem is, the tests ran fine even when I removed the code I added in `vote_views` that prevented voting on "schedule published" conference proposals. See https://github.com/smtchahal/junction/commit/ee3b7c578d4a12343f14aaf14ad37a2ebaeec41f.

**Pull request merged**